### DISCUSSION TITLE
How to deploy from a local docker image?

### DISCUSSION THREAD

**Discussion by IgnisDa** [dokku/dokku#5486] [Q&A]
Earlier this was possible: https://dokku.com/docs~v0.8.2/deployment/methods/images/#deploying-an-image-from-ci

But `dokku:tags` has since been removed. `dokku git:from-image` does not respect locally present images.

Is it possible to deploy local docker images?

**Comment by josegonzalez** [comment]
It should respect a locally present image since all it does is create a dockerfile with a `FROM` statement. It sounds like a bug that it doesn't, so I'm transferring this to a ticket. Please add replication steps + the output of `dokku report` so I can see whats going on.

See #5492.
